scripts/scr_vehicle_order/scr_vehicle_order.gml

```diff
@@ -1,159 +1,70 @@
-			temp_wep3[co,vehicle]=veh_wep3[co,i];
-			temp_upgrade[co,vehicle]=veh_upgrade[co,i];
-			temp_acc[co,vehicle]=veh_acc[co,i];
-			temp_hp[co,vehicle]=veh_hp[co,i];
-			temp_chaos[co,vehicle]=veh_chaos[co,i];
-			temp_uid[co,vehicle]=veh_uid[co,i];
-	    }
-	}
-	for(var i=1; i<=150; i++){
-	    if (veh_role[co,i]="Whirlwind"){
-			vehicle+=1;
-			temp_race[co,vehicle]=veh_race[co,i];
-			temp_loc[co,vehicle]=veh_loc[co,i];
-			temp_name[co,vehicle]=veh_name[co,i];
-			temp_role[co,vehicle]=veh_role[co,i];
-			temp_lid[co,vehicle]=veh_lid[co,i];
-			temp_wid[co,vehicle]=veh_wid[co,i];
-			temp_wep1[co,vehicle]=veh_wep1[co,i];
-			temp_wep2[co,vehicle]=veh_wep2[co,i];
-			temp_wep3[co,vehicle]=veh_wep3[co,i];
-			temp_upgrade[co,vehicle]=veh_upgrade[co,i];
-			temp_acc[co,vehicle]=veh_acc[co,i];
-			temp_hp[co,vehicle]=veh_hp[co,i];
-			temp_chaos[co,vehicle]=veh_chaos[co,i];
-			temp_uid[co,vehicle]=veh_uid[co,i];
-	    }
-	}
-	for(var i=1; i<=150; i++){
-	    if (veh_role[co,i]="Land Speeder"){
-			vehicle+=1;
-			temp_race[co,vehicle]=veh_race[co,i];
-			temp_loc[co,vehicle]=veh_loc[co,i];
-			temp_name[co,vehicle]=veh_name[co,i];
-			temp_role[co,vehicle]=veh_role[co,i];
-			temp_lid[co,vehicle]=veh_lid[co,i];
-			temp_wid[co,vehicle]=veh_wid[co,i];
-			temp_wep1[co,vehicle]=veh_wep1[co,i];
-			temp_wep2[co,vehicle]=veh_wep2[co,i];
-			temp_wep3[co,vehicle]=veh_wep3[co,i];
-			temp_upgrade[co,vehicle]=veh_upgrade[co,i];
-			temp_acc[co,vehicle]=veh_acc[co,i];
-			temp_hp[co,vehicle]=veh_hp[co,i];
-			temp_chaos[co,vehicle]=veh_chaos[co,i];
-			temp_uid[co,vehicle]=veh_uid[co,i];
-	    }
-	}
-	for(var i=1; i<=150; i++){
-	    if (veh_role[co,i]="Land Raider"){
-			vehicle+=1;
-			temp_race[co,vehicle]=veh_race[co,i];
-			temp_loc[co,vehicle]=veh_loc[co,i];
-			temp_name[co,vehicle]=veh_name[co,i];
-			temp_role[co,vehicle]=veh_role[co,i];
-			temp_lid[co,vehicle]=veh_lid[co,i];
-			temp_wid[co,vehicle]=veh_wid[co,i];
-			temp_wep1[co,vehicle]=veh_wep1[co,i];
-			temp_wep2[co,vehicle]=veh_wep2[co,i];
-			temp_wep3[co,vehicle]=veh_wep3[co,i];
-			temp_upgrade[co,vehicle]=veh_upgrade[co,i];
-			temp_acc[co,vehicle]=veh_acc[co,i];
-			temp_hp[co,vehicle]=veh_hp[co,i];
-			temp_chaos[co,vehicle]=veh_chaos[co,i];
-			temp_uid[co,vehicle]=veh_uid[co,i];
-	    }
-	}
-	// Return here
-	for(var i=1; i<=150; i++){
-	    if (i<=vehicle){
-	        veh_race[co,i]=temp_race[co,i];
-	        veh_loc[co,i]=temp_loc[co,i];
-	        veh_name[co,i]=temp_name[co,i];
-	        veh_role[co,i]=temp_role[co,i];
-	        veh_lid[co,i]=temp_lid[co,i];
-	        veh_wid[co,i]=temp_wid[co,i];
-	        veh_wep1[co,i]=temp_wep1[co,i];
-	        veh_wep2[co,i]=temp_wep2[co,i];
-	        veh_wep3[co,i]=temp_wep3[co,i];
-	        veh_upgrade[co,i]=temp_upgrade[co,i];
-	        veh_acc[co,i]=temp_acc[co,i];
-	        veh_hp[co,i]=temp_hp[co,i];
-	        veh_chaos[co,i]=temp_chaos[co,i];
-	        veh_uid[co,i]=temp_uid[co,i];
-	    }
-	    if (i>vehicle){
```

Comment:
this whole if is not needed, because we already assign default values during initialization